modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -190,28 +190,28 @@ The chart on the *Application Network* tab shows the xref:visualizer::troublesho
 The following charts appear on the *Overview* dashboard:
 
 Inbound - Total Requests by Response Type::
-The sum of inbound requests, grouped by type of response in automatic intervals. Responses are `OK` or `Failed`. If a Mule app has more than one connector type, all types are included in this metric.
+The sum of inbound requests, grouped by type of response by automatic interval. Responses are `OK` or `Failed`. If a Mule app has more than one connector type, all types are included in this metric.
```

Comment:
Are the requests grouped by type or interval or both? Is it one sum but displayed by type or interval? Or is it all the OK ones in an interval, and all the Failed ones in an interval?
It might be easier to say 
```suggestion
The sum of all `OK` and all `Failed` inbound requests during an interval. If a Mule app has more than one connector type, all types are included in this metric.
```
I'm also unclear how the second sentence ties in to the first, mostly because we use "type" again, but in a different way.

Reply:
I would like to hear more about this; I'm really confused by it.